pyanaconda/ui/tui/spokes/software_selection.py

```diff
@@ -303,7 +303,7 @@ def ready(self):
         return not threadMgr.get(THREAD_SOFTWARE_WATCHER) \
             and not threadMgr.get(THREAD_PAYLOAD) \
             and not threadMgr.get(THREAD_CHECK_SOFTWARE) \
-            and self.payload.base_repo is not None
+            and ((self.payload.base_repo is not None) or self.errors)
```

Comment:
I am afraid this means that the Software Selection screen can be now entered even if the installation source is not properly set up. That can have unexpected consequences.

Reply:
@jstodola @vslavik that could be problematic. Could you please take a look on that?

Reply:
I will, once I know what to do about it...

Reply:
I didn't encounter any unexpected behavior with anaconda-34.25.1.11-1.el9 (which includes this change) when the installation source is not set or when it is set incorrectly (like having an unusable URL). The Software selection spoke just complains that "Installation source needs to be set up first." - so the same behavior like in RHEL-9.0.

Reply:
Oh, right. The Software Selection screen can actually survive that in TUI. Let's leave it as it is in that case.